src/sentry/api/endpoints/organization_event_details.py

```diff
@@ -54,17 +54,22 @@ def get(self, request, organization, project_slug, event_id):
                 event=event,
                 query=request.query_params.get("query"),
                 params=params,
+                organization=organization,
                 reference_event=reference,
                 referrer="api.organization-event-details",
             )
         except discover.InvalidSearchQuery as err:
             raise ParseError(detail=six.text_type(err))
 
         data = serialize(event)
-        data["nextEventID"] = pagination.next
-        data["previousEventID"] = pagination.previous
-        data["oldestEventID"] = pagination.oldest
-        data["latestEventID"] = pagination.latest
+        data["nextEventID"] = pagination.next.event_id
```

Comment:
The discover2 ui is the only code that uses this endpoint. We could return formatted slugs to reduce work in the ui.